models/staging/stg_dbt_run_results.sql

```diff
@@ -0,0 +1,45 @@
+with base as (
+
+    select * from {{ ref('stg_dbt_artifacts') }}
+
+),
+
+run_results as (
+
+    select * from base
+    where artifact_type = 'run_results.json'
+
+),
+
+dbt_run as (
+
+    select * from run_results
+    where data:args:which = 'run'
+
+),
+
+fields as (
+
+    select
+        generated_at as artifact_generated_at,
+        data:metadata:invocation_id::string as command_invocation_id,
```

Comment:
i dont think we need to reparse this in this model since the `command_invocation_id` is already parsed in `stg_dbt_artifacts`